apps/src/templates/codeDocs/MethodSummaryTable.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import EnhancedSafeMarkdown from '@cdo/apps/templates/EnhancedSafeMarkdown';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import color from '@cdo/apps/util/color';
+import {tableLayoutStyles} from '@cdo/apps/templates/tables/tableConstants';
+import {TextLink} from '@dsco_/link';
+
+export default function MethodSummaryTable({
+  methods,
+  programmingClassLink,
+  openLinkInNewTab = false
```

Comment:
I imagine that for the instructions panel we would want the link to open in a new tab but when this is added to the programming class page, we wouldn't. If it seems like overkill to add this prop now I can remove it.

Reply:
I think it makes sense to add it now!